networks/north-america/us/ma/mbta.json

```diff
@@ -0,0 +1,67 @@
+{
+  "public_transport/platform/bus|MBTA Bus Stop": {
```

Comment:
If we have a separate entry for each kind of feature that can be associated with the network, then there will be a lot of duplication, not just between the various entries in this file but also among each file. It would also become tedious to add support for associating a network with, say, ticket vending machines with networks after the fact.

Instead of orienting these files around presets, how about focusing on networks and indicating the applicable modes of transportation as fields on each network?

Reply:
Could you go into more detail? Do you mean like defining "this is the network and then these are the things that can be part of this network"?

Reply:
Yes. For example, most county transportation agencies would be defined by a single entry describing the network. There would be a `bus` property set to true, and elsewhere NSI or iD would take that to mean that the network should apply to the existing Bus Stop, Bus Station, and Bus Route presets. If the same agency operates a bus rapid transit line that’s part of the same network, with ticket vending machines along the route, then setting `vending_machine` to true would automatically associate the network with that preset too.

That’s the simple case, anyways. I’m sure there are edge cases that maybe we can address by specifying overrides of individual tags.